src/base/memory_monitor_cl.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_cl.hpp - client structures and functions
+ *                         for memory monitoring module
+ */
+
+#ifndef _MEMORY_MONITOR_CL_HPP_
+#define _MEMORY_MONITOR_CL_HPP_
+
+#include <cstdint>
+
+#include "dbtype_def.h"
+#include "memory_monitor_common.h"
+
+const char *module_names[] =
+{
+  "",				/* dummy */
```

Comment:
Why id the dummy needed. A comment explaining it is needed if it's necessary.

Reply:
It's because of aligning module index what utility and server provides. I add the reason of why dummy needed in 334d324.

Reply:
I mean why it has to start from 1? Can't it start from 0? It seems simpler and doesn't need "dummy".

Reply:
As mentioned before, the indexes of the module array will remain consistent with the indexes entered by the user in the memmon utility. And in the diagdb utility, the indexes also start from 1. Therefore, we have decided to maintain this consistency.

Reply:
This reply also serves as an reply to the following comment: https://github.com/CUBRID/cubrid/pull/4447#discussion_r1243019747